pymongo/client_session.py

```diff
@@ -947,9 +947,18 @@ def _txn_read_preference(self):
             return self._transaction.opts.read_preference
         return None
 
+    def _start_serv_sess(self):
+        if isinstance(self._server_session, _EmptyServerSession):
+            self._server_session, old = (
+                self._client._topology.get_server_session(),
+                self._server_session,
+            )
+            for _ in range(old.started):
```

Comment:
This should be an if-statement instead of a loop (see by other comment below.

Reply:
Done.